gordon/plugins_loader.py

```diff
@@ -82,7 +82,7 @@ def _init_plugins(active_plugins, installed_plugins, plugin_configs):
             continue
 
         try:
-            plugin_object = plugin_class(plugin_config)
+            plugin_object = plugin_class(plugin_config, **kwargs)
             inited_plugins.append(plugin_object)
             inited_plugin_names.append(plugin_name)
         except Exception as e:
```

Comment:
Can this be more specific?

Reply:
Not really, we do want to catch any exception that loading a plugin might throw.

Reply:
Is there anything we should reraise, then?

Reply:
Depends on if the user sets the `debug = True` in config. It's handled in `main:_log_or_exit_on_exceptions`